docs/uml-class-diagrams/database/production/DatabaseGisInterfaceImplementation/DatabaseGisInterfaceImplementation.puml

```diff
@@ -2,7 +2,16 @@
 
 package edu.oakland.production.database {
     class DatabaseGisInterfaceImplementation {
+        - DatabaseGisManager manager
+        - LocationDataPoint locDataPoint
+        - String currentSatellite
+        - String mode
```

Comment:
Does this class need to store these private variables? If not, make sure you remove them.
```suggestion
        - LocationDataPoint locDataPoint
        - String currentSatellite
        - String mode
```